hapi-fees/src/main/java/com/hedera/services/usage/crypto/CryptoOpsUsage.java

```diff
@@ -51,6 +50,9 @@
 	private static final long LONG_BASIC_ENTITY_ID_SIZE = BASIC_ENTITY_ID_SIZE;
 	private static final long LONG_ACCOUNT_AMOUNT_BYTES = USAGE_PROPERTIES.accountAmountBytes();
 
+	static final long CREATE_SLOT_MULTIPLIER = 1228;
+	static final long UPDATE_SLOT_MULTIPLIER = 24000;
```

Comment:
Where are these values from?

Reply:
haha, @tinker-michaelj and I would like to say hours of market research. But its just a number to get us close to $0.0018 / autoAssociation-slot price

Reply:
So they are constants that are reached by try-and-fail based on current context. If any condition changes, these values need re-calculation. We may need put explanations on how to get these numbers for future references.